lib/build.js

```diff
@@ -8,7 +8,16 @@ const pify = require('pify');
 const writeFile = pify(fs.writeFile);
 
 module.exports = context => {
-	const dockerfile = `FROM node:${context.version}-onbuild`;
+	const dockerfile = [
+		`FROM node:${context.version}`,
+		'RUN mkdir -p /usr/src/app',
+		'WORKDIR /usr/src/app',
+		'COPY package.json /usr/src/app/',
+		'RUN npm install && npm cache clean --force',
```

Comment:
What's `npm cache clean` for?

Reply:
That's just clearing npm's cache before Docker commits that image layer from my understanding. It's probably not strictly necessary since the image is going to be deleted soon after creation. But I added it to emulate the behavior of the last available node-onbuild [Dockerfile](https://github.com/nodejs/docker-node/blob/master/8/onbuild/Dockerfile#L9).